docs/change-controlled-docs.rst

```diff
@@ -0,0 +1,233 @@
+This page describes the development and release processes of change-controlled documents produced by DM.
+
+.. _CCB: https://project.lsst.org/groups/ccb/
+.. _Document-9224: https://ls.st/Document-9224
+.. _lsst-texmf: https://lsst-texmf.lsst.io
+.. _LPM-19: https://ls.st/LPM-19
+.. _LPM-51: https://ls.st/LPM-51
+.. _GitHub: https://github.com/lsst
+.. _LDM-294: https://ls.st/LDM-294
+
+.. _ccd-regular:
+
+Regular development
+===================
+
+Develop change-controlled documents using :doc:`DM's standard workflow <../processes/workflow>`.
+That is:
+
+1. Create a ticket branch from ``master``.
+2. Work on the ticket branch.
+3. Have the ticket peer-reviewed.
+4. Rebase and merge the ticket branch to ``master``.
+
+Merging to ``master`` does not denote acceptance by a CCB.
+Instead, it adds to the changeset that will be included in the next CCB review.
+
+.. figure:: ccd-develop.svg
+   :name: fig-ccd-develop
+   :align: center
+   :target: ccd-develop.svg
+   :alt: Git commit tree showing two ticket branches merged to master.
+
+   Example of Git branches during document development.
+   Staff write and edit the document on individual ticket branches that they merge to master, following DM's regular code workflow.
+
+.. _ccd-submit:
+
+Submitting a document to the CCB
+================================
+
+Establishing a new baselined version of the document begins with submitting it to the CCB.
+The release manager is responsible for orchestrating this process on behalf of DM.
+
+1. Choose a commit on ``master`` where the document's content is ready for CCB review.
+2. Create a DocuShare tag at the commit.
+   DocuShare tags are formatted as ``docushare-vN``, where ``N`` is the next-available document version number in DocuShare.
+3. Upload the PDF built by CI for the ``docushare-vN`` tag to DocuShare.
+   By definition, this uploaded document should become version ``N`` in DocuShare.
+4. For LDM documents, create a DM RFC.
+   For project documents (LSE), create an LCR.
+5. Once the RFC or LCR number is known, create a branch from the ``docushare-vN`` tag named ``tickets/RFC-N`` or ``tickets/LCR-N``.
+   This release branch initially has no new commits on it.
+
+.. figure:: ccd-submit.svg
+   :name: fig-ccd-submit
+   :align: center
+   :target: ccd-submit.svg
+   :alt: Git commit tree showing a docushare-v1 tag added to the head of the master branch.
+
+   Example Git tree showing document submission to the CCB.
+   The release manager creates a ``docushare-v1`` tag from the head of the ``master`` branch, since ``1`` is the next-available version number for the document in DocuShare.
```

Comment:
It is highly unlikely that 1 will ever be used because when you first create a document in docushare for an LDM you don't usually have a document to upload to you ask docushare to create a stub. That will be v1.